src/fragile-goods/lib/logic.ergo

```diff
@@ -15,7 +15,7 @@
 namespace io.clause.demo.fragileGoods
 
 contract FragileGoods over FragileGoodsClause {
-  clause fragilegoods(request : DeliveryUpdate) : PayOut {
+  clause fragilegoods(request : DeliveryUpdate) : PayOut emits org.accordproject.cicero.runtime.PaymentObligation {
```

Comment:
Do we support `import` so that you don't have to use a FQN?

Reply:
Yes, this is in Ergo 0.1.1. I'll update the template once we get that into Cicero